src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -80,6 +81,28 @@ export interface IQuickPickTerminalObject {
 	keyMods: IKeyMods | undefined
 }
 
+export interface TerminalCommand {
+	command: string;
+	timestamp: number;
+	getOutput: () => string | undefined;
+	cwd?: string;
+	exitCode?: number;
+	marker?: IMarker;
```

Comment:
nit: keep functions on bottom and use `f(): return` syntax

```suggestion
	cwd?: string;
	exitCode?: number;
	marker?: IMarker;
	getOutput(): string | undefined;
```